src/NUnitEngine/nunit.engine/Services/ResultService.cs

```diff
@@ -67,6 +77,10 @@ public IResultWriter GetResultWriter(string format, object[] args)
                     return new NUnit3XmlResultWriter();
                 case "cases":
                     return new TestCaseResultWriter();
+#if NETSTANDARD1_3
+                default:
+                    return null;
+#else
                 case "user":
                     return new XmlTransformResultWriter(args);
```

Comment:
None of the XML transform classes we use are available and they aren't used in the VS Adapter.